test/test_docker_build_direct.py

```
@@ -0,0 +1,53 @@
+import unittest
+
+import docker
+from exasol_script_languages_container_tool.cli.commands import build, clean_flavor_images
+
+from exasol_script_languages_container_tool.lib.utils.docker_utils import find_images_by_tag
+from click.testing import CliRunner
+
+import utils as exaslct_utils
+from exasol_integration_test_docker_environment.testing import exaslct_test_environment
+
+
+class DockerBuildDirectTest(unittest.TestCase):
+
+    def setUp(self):
+        print(f"SetUp {self.__class__.__name__}")
+        self.test_environment = exaslct_test_environment.ExaslctTestEnvironment(self, exaslct_utils.EXASLCT_DEFAULT_BIN)
+        self.docker_client = docker.from_env()
+        self.runner = CliRunner()
+        self._clean_images()
+
+    def tearDown(self):
+        try:
+            self.docker_client.close()
+        except Exception as e:
+            print(e)
+        self._clean_images()
```

why eat the exception?

---

I changed the logic.